src/commands.ts

```diff
@@ -38,6 +41,10 @@ const commands = {
             task.definition.args[0] === 'package' &&
             task.definition.args[1] === 'resolve'
         )!;
+        task.presentationOptions = {
+            reveal: vscode.TaskRevealKind.Silent
```

Comment:
If we get the problem matcher working we can default the resolve task to silent i.e. set the presentation option at creation

Reply:
I was thinking if perhaps we should treat user-initiated tasks separately, and never silence them.

I also saw #31, but I don't know of a way to apply that to user-initiated tasks, so they may be treated differently there as well?

Reply:
Might as well set the `presentationOptions` when the resolve task is created in SwiftTaskProvider.ts

Reply:
Can we add this to the update dependencies task as well?

Reply:
One issue I see with adding `Silent` as the default presentation is that users who trigger the task from the **Run Task...** menu will not get much feedback when they start the task. It may be better to stick to the default presentation (pop up the terminal) when the user actively initiates the task?

For the update command: we could do this, but this command is currently only used for the Update Dependencies button in the Dependencies view, so it's also user-initiated.

Reply:
There is no reason for the user to trigger resolve from the "run task..." menu, so I don't think this is an issue. Anyway there is only one copy of the resolve task and it runs straight off and you set it to silent. So it would be silent from then on regardless of how it was triggered.
 
The update command it is just a nicety. We have the status item we might as well use it.

Reply:
OK, I'll add it for the update command.

I did notice that the task provider runs every time we call `vscode.tasks.fetchTasks()` to launch a command. So every command does run a different task object, with its own presentation settings.

Reply:
It doesn't always call run the task provider I've found.

Reply:
One last thing can we move the `start` to before the `try` and the `end` after the catch block. If the `executeTaskAndWait` should throw an error (not that it should) `statusItem.end` would never get called

Reply:
Can anything other than `executeAndWait` throw an error? I'd need to move the `try` down a bit because I need the `task` object.

Reply:
@svanimpe just skip that. It won't work as expected. I think this is good to go once the update command is done.